oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/FVShared.java

```diff
@@ -234,25 +244,22 @@ private void updateActiveKeyboardsList() {
                 KMManager.removeKeyboard(context, i);
         }
 
+        File resourceRoot =  new File(getResourceRoot());
+        PackageProcessor kmpProcessor =  new PackageProcessor(resourceRoot);
+
         // Recreate active keyboards list
         for(FVRegion region : regionList) {
             for(FVKeyboard keyboard : region.keyboards) {
                 if(loadedKeyboards.contains(keyboard.id)) {
-                    // Load the .keyboard_info file and find its first language code
-                    Keyboard kbInfo = new Keyboard(
-                      FVDefault_PackageID, //TODO: we want to share keyboard build scripts between ios and android; can we do this?
+                    // Parse kmp.json for the keyboard info
+                    Keyboard kbd = kmpProcessor.getKeyboard(
+                      FVDefault_PackageID,
                       keyboard.id,
-                      keyboard.name,
-                      "en", //TODO: use language code from kmp.json
-                      keyboard.name,
-                      "1.0", //TODO: use keyboard version from kmp.json
-                      String.format("%s%s", FVKeyboardHelpLink, keyboard.id),
-                      "", // kmp link
-                      false,
-                      "NotoSansCanadianAboriginal.ttf",
-                      "NotoSansCanadianAboriginal.ttf");
-
-                    KMManager.addKeyboard(context, kbInfo);
+                      null); // get first associated language ID
+                    if (kbd != null) {
+                      // TODO: Override fonts to NotoSansCanadianAboriginal.ttf
```

Comment:
atm, kmp.json doesn't specify font info for the touch layout keyboards.
Should we 
1. override the font here
2. fix the fv_all build script to use "NotoSansCanadianAboriginal.ttf"

Reply:
I think we should fix the fv_all build script to use the fonts specified in the other .kps files -- which may differ per keyboard, I haven't checked.

Reply:
So searching through the fv/*.kps files:

0 use NotoSansCanadianAboriginal.ttf
33 *include* shared Aboriginal*.ttf fonts, but the touch layout keyboards don't specify font info...